TesseractOCR/G8Tesseract.mm

```diff
@@ -51,6 +51,22 @@ @implementation G8Tesseract
 
 @synthesize absoluteDataPath=_absoluteDataPath;
```

Comment:
You could write ```@property (..., readonly, ...) ...``` in header and ```@property (..., readwrite, ...) ...``` or just ```@property (..., ...) ...``` in private category for that kind of properties.